dev-python/colorcet/colorcet-2.0.6.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} pypy3 )
+
+inherit distutils-r1
+
+DESCRIPTION="Collection of perceptually uniform colormaps"
+HOMEPAGE="
+	https://pypi.org/project/colorcet
+	https://github.com/holoviz/colorcet"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
```

Comment:
We can drop [empty](https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-620007.3) IUSE (this is a nit, I just like giving people the link to PMS so they're aware of it - it's ok to keep it as-is).

Reply:
Nice! Nice nitpick, too. I'm all for dropping extraneous lines.